app/gen-server/lib/homedb/HomeDBManager.ts

```diff
@@ -3646,12 +3655,14 @@ export class HomeDBManager extends EventEmitter {
     const max =
       billingAccount.getFeatures().maxNewUserInvitesPerOrg ??
       Deps.DEFAULT_MAX_NEW_USER_INVITES_PER_ORG;
-    const current = await this.getNewUserInvitesCount(orgKey, {
-      excludedUserIds: newUsers.map((user) => user.id),
-      transaction: manager,
-    });
-    if (current + delta > max) {
-      throw new ApiError("Your site has too many pending invitations", 403);
+    if (max !== undefined) {
```

Comment:
We can move this to the beginning of the method and do an early return if `max` is undefined.